src/Tags/Vite.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Statamic\Tags;
+
+use Illuminate\Foundation\Vite as LaravelVite;
+
+class Vite extends Tags
+{
+    /**
+     * The {{ vite }} tag.
+     *
+     * @return string|array
+     */
+    public function index()
+    {
+        if (! $src = $this->params->get('src')) {
+            throw new \Exception('Please provide a source file.');
```

Comment:
Also, I don't think you _need_ to provide a source. Looking at the code, it doesn't seem to care if you pass it nothing.